.github/workflows/build-deb-package.yml

```diff
@@ -1,90 +1,78 @@
-name: "Build Packages"
-on:
-  push
-
+---
+name: Build Packages
+on: push
 jobs:
   build_deb:
-    name: "Build ${{ matrix.os }} Package"
```

Comment:
I honestly don't think we should encourage string in yaml without quote, is too error prone.

Reply:
We've decided to add `preserve_quotes = true` in yamlfix config to avoid this issue